### ISSUE TITLE
Bug: Additional `_` being added to import of `__`

### ISSUE THREAD

**Issue by TylorS** [peterjuras/vsc-sort-imports#14]
## To Reproduce
Create a file `foo.ts` and add 
```typescript
import { __ } from 'ramda'
```

## Expected Results

Sorts imports as usual

## Actual Results

With each time I save an additional `_` is appended to the import

1 save 
```typescript
import { ___ } from 'ramda'
```
2 saves 
```typescript
import { ____ } from 'ramda'
```
and so on

Thank you very much for this very useful extension, it's been indispensable to my team and I!

**Comment by peterjuras** [comment]
Thanks for raising this issue, this is quite a weird bug! It only affects TypeScript files, JavaScript files are fine.

I raised [an issue](https://github.com/renke/import-sort/issues/18) in the `import-sort` repository which takes care of TypeScript parsing and output, if I have time I will look into it myself!

**Comment by peterjuras** [comment]
This bug is fixed now thanks to @renke in v2.4.0
